src/afa/afa.cc

```diff
@@ -14,6 +14,10 @@
  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
  * GNU General Public License for more details.
  */
+#pragma clang diagnostic push
```

Comment:
Maybe we could add a comment above these explaining that AFA is in unmaintained state as of now.

Reply:
Done